src/site/filters/liquid.js

```diff
@@ -36,6 +36,37 @@ module.exports = function registerFilters() {
     return dt;
   };
 
+  liquid.filters.formatVaParagraphs = vaParagraphs => {
+    const FIRST_SECTION_HEADER = 'VA account and profile';
+    const LAST_SECTION_HEADER = 'Other topics and questions';
+
+    // Derive the first and last sections.
+    const firstSection = _.find(
+      vaParagraphs,
+      vaParagraph =>
+        vaParagraph.entity.fieldSectionHeader === FIRST_SECTION_HEADER,
+    );
+    const lastSection = _.find(
+      vaParagraphs,
+      vaParagraph =>
+        vaParagraph.entity.fieldSectionHeader === LAST_SECTION_HEADER,
+    );
+
+    const otherSections = _.filter(
+      vaParagraphs,
+      vaParagraph =>
+        vaParagraph.entity.fieldSectionHeader !== FIRST_SECTION_HEADER &&
+        vaParagraph.entity.fieldSectionHeader !== LAST_SECTION_HEADER,
+    );
+
+    return [
+      firstSection,
+      // Other sections is sorted alphabetically by `fieldSectionHeader`.
+      ..._.orderBy(otherSections, 'entity.fieldSectionHeader', 'asc'),
+      lastSection,
+    ];
+  };
```

Comment:
This definitely seems specific to the R&S section...wondering if we can indicate that in the name of the filter or something. I'm also wondering if we can add a unit test for this in the sibling file `liquid.unit.spec.js`?

Reply:
Indeed, that's what I warned about [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/15649#issuecomment-756216700) actually. Since this gets run for all lists_of_links, I think giving it an R&S section variable name is misleading since it will technically be run on all lists_of_links 😬

Reply:
Oh geez I just realized that we are making changes to the shared template `src/site/paragraphs/lists_of_links.drupal.liquid`... that would apply the toggle behavior across the whole website. Does that seem to only affect the R&S homepage?

Reply:
For now yes, but that's a big problem, I have no way in the lists_of_links to know if it's coming from R&S homepage atm... uggh